Add Release Compatibility Chart

inc1t3Ful opened the pull request:
++ ReleaseCompatibility Chart
** R2.0 Upgrade Guide updated to mention official release and changes
** fixed many formatting issues on with previous docs not showing lists/ tables/ code blocks

arvindsree commented on website/docs/r/aviatrix_fqdn.html.markdown:
```diff
@@ -0,0 +1,48 @@
+* `gw_list` - (Optional) Name of the gateway. One or more gateways as list ["gw1", "gw2"]
+* `domain_list` - (Optional) One or more domain names in a list with details as listed below
+    * `fqdn` - (Optional) FQDN. Example: "facebook.com" 
+    * `proto` - (Optional) Protocol. Example "tcp" 
```
Could you add other valid values supported here, I think they are "icmp" and "udp"

inc1t3Ful committed:
Add Release Compatibility Chart

++ ReleaseCompatibility Chart
** R2.0 Upgrade Guide updated to mention official release and changes
** fixed many formatting issues on with previous docs not showing lists/ tables/ code blocks
